spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8SecretsPropertySourceTest.java

```diff
@@ -41,23 +37,18 @@
 
 import static java.util.Collections.singletonMap;
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatNoException;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 @ExtendWith(SpringExtension.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = App.class,
 		properties = "spring.main.cloud-platform=KUBERNETES")
 @TestPropertySource("classpath:/application-secrets.properties")
 @EnableKubernetesMockClient(crud = true, https = false)
-public class Fabric8SecretsPropertySourceTest {
+class Fabric8SecretsPropertySourceTest {
 
 	private static final String NAMESPACE = "test";
 
 	private static KubernetesClient mockClient;
 
```

Comment:
I want to remove `@ClassRule` and because of that this test needs slight changes, because of the way `@EnableKubernetesMockClient` works. This does not change the logic of the test at all. It does two things:

- removes `@ClassRule` and as such slightly changes
- moves two tests into `Fabric8SecretsPropertySourceMockTests` (where I will add more tests soon)